packages/shared/src/modules/users/libs/types/user-info-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type UserInfoResponseDto } from "./user-info-response-dto.type.js";
+
+type UserInfoRequestDto = UserInfoResponseDto;
```

Comment:
Rename to UserPatchRequestDto
```suggestion
type UserPatchRequestDto = {
  name: string;
};
```